### PR TITLE
Kalman filter alert system project documentation

### PR DESCRIPTION
**Pull request by economuteo**
### Pull Request Overview

This pull request adds the photos with the project and everything that was needed.

### Testing Strategy

This pull request was tested by...

### TODO or Help Wanted

This pull request still needs a veriffication

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: 
Economu Teo-Antonio <economuteo@gmail.com>
Fotescu Maria-Irina <fotescufmi03@yahoo.com>

### CODE REVIEW

**Comment by kaanmeister** [comment]
Fill the pull request template and KiCad folder has to be in your project repository not here.

**Comment by alexandruradovici** [comment]
Please fix the conflicts.

**Comment by alexandruradovici** [comment]
Please fill the pull request template.

**Comment by economuteo** [comment]
> Please fill the pull request template.

Is it ok now? I filled it!

File: .anima/.gitignore
Comment (alexandruradovici) [inline .anima/.gitignore:1]: Please delete this.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: .anima/.gitignore
Comment (economuteo) [reply .anima/.gitignore:1]: I deleted it

Comment (alexandruradovici) [reply .anima/.gitignore:1]: The .idea folder is still here.

**Review by ZamfirAndreiPaul — APPROVED** [review]

File: .anima/.gitignore
Comment (economuteo) [reply .anima/.gitignore:1]: > The .idea folder is still here.

I checked every folder in my project and now everything seems ok. I committed everything again. I hope it's ok now!

Comment (economuteo) [reply .anima/.gitignore:1]: I wrote you a message on Teams about a problem that I'm facing right now.

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (alexandruradovici) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Please do not edit this file. This is the template that will be added to all pull requests' descriptions. Please edit the pull request description in github's interface.

File: docs/project/index.md
Comment (alexandruradovici) [inline docs/project/index.md:1]: Please do not edit this file.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (alexandruradovici) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Please reset the file to the original.

File: docs/project/2024/maria-irina.fotescu_teo-antonio.economu/index.md [hunk 1]
```diff
@@ -6,7 +6,6 @@ _Author_: Fotescu Maria-Irina, Economu Teo-Antonio
 
 _GitHub Project Link_: https://github.com/UPB-FILS-MA/project-economuteo
 
-:::
```
Comment (alexandruradovici) [inline docs/project/2024/maria-irina.fotescu_teo-antonio.economu/index.md:9]: Please do not delete this line.

File: docs/project/index.md
Comment (alexandruradovici) [inline docs/project/index.md:1]: Please reset this file to the original.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Comment by Fote20003** [comment]
Is it ok, now?

**Comment by economuteo** [comment]
Is it ok now?

**Comment by alexandruradovici** [comment]
Please take a look at the changed files, the files still have modifications. Please copy and paste the original files.

**Comment by Fote20003** [comment]
Is it ok now?

**Pull request closed without merging**